py-polars/polars/datatypes/classes.py

```diff
@@ -552,11 +554,18 @@ class Enum(DataType):
     Parameters
     ----------
     categories
-        The categories in the dataset. Categories must be strings.
+        The categories in the dataset. Categories must be strings or a `StrEnum`.
     """
 
     categories: Series
 
+    if sys.version_info >= (3, 11):
```

Comment:
I was originally going to constantize this, but it seems doing that impacts the conditional overload resolutions supported by type-checkers: https://github.com/pola-rs/polars/pull/19725/commits/d5b23ab5c319d9c5c7225e939c4dfa67622c6e59

That being, using a constant means a type-checker like Pyright could resolve this as the overload for thoughts below 3.11, which we want to avoid.